synapse/handlers/room.py

```diff
@@ -285,6 +285,7 @@ def clone_existing_room(
             (EventTypes.RoomAvatar, ""),
             (EventTypes.Encryption, ""),
             (EventTypes.ServerACL, ""),
+            (EventTypes.Member, None),
```

Comment:
I think it would probably be better to do this with a separate call to `get_filtered_current_state_ids`.

Reply:
Hrm, in testing this seems to be a few hundred ms slower if we do another call to the DB.

Reply:
how big was your room? ;-p

Reply:
Literally 1 ban and no messages lol